### PR TITLE
Task document

### PR DESCRIPTION
**Pull request by georginaw**

### CODE REVIEW

File: functions.php [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+require_once ('index.php');
+
+/*
+ * displays each film and its attributes (from the database) in a human readable way
+ *
+ * @param array which is an array containing arrays for each film
+ *
+ * return string displaying films and film details
+ */
+function displayFilms(array $array) : string {
```
Comment (DanielGardiner) [inline functions.php:12]: function states it returns a string, but actually returns nothing (no return statement) and instead echos

File: functions.php [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+    foreach ($array as $item) {
+        echo '<h3 style="margin-bottom: 5px">' . $item['title'] . '</h3>';
+        echo '<p style="margin:0">' . 'Year released: ' . $item['release_year'] . '</p>';
```
Comment (DanielGardiner) [inline functions.php]: using style tags inline, instead put in separate CSS file to avoid repeating?

**Review by DanielGardiner — CHANGES_REQUESTED** [review]

File: functions.php (hunk 1 above)
Comment (georginaw) [reply functions.php:12]: Ok, edited and sorted

File: functions.php (hunk 2 above)
Comment (georginaw) [reply functions.php]: Ok, added CSS and put in some html to link it up, got rid of inline styles.

**Review by Noctiscreo — APPROVED** [review]
Looks good to me!

**Pull request merged**